app/builders/ndr_ui/bootstrap/readonly.rb

```diff
@@ -51,6 +51,12 @@ def grouped_collection_select(method, collection, group_method, group_label_meth
             @template.content_tag(:p, readonly_value, class: 'form-control-static')
           end
 
+          # For labels, the for attribute should be removed:
+          def label(method, text = nil, **options, &block)
+            return super unless readonly?
+            super(object_name, method, (options || {}).merge(for: nil), &block)
```

Comment:
`**options` will always yield a hash, I think, so this can just be `options.merge(for: nil)` ?